installation/resources/installer-local.yaml

```
@@ -63,6 +63,149 @@ spec:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: helm-certs-sa
```

Nit: Maybe different SA name
```suggestion
  name: helm-certs-job-sa
```